### ISSUE TITLE
Would this work on a Windows host using Hyper-V/WHPX as qemu does? 

### ISSUE THREAD

**Issue by Nislaco** [tenclass/mvisor#22]
Sorry to have to ask, I do notice it looks like the dependencies are available for windows host/Msys2. Please and thanks.

**Comment by 78** [comment]
No. KVM can only work on Linux. Mvisor uses KVM to maximize the performance.
WHPX can be an option in the future. @nooodles2023 consider this.
